src/python-frontend/python_converter.cpp

```diff
@@ -670,6 +667,21 @@ exprt python_converter::get_function_call(const nlohmann::json &element)
       return rhs;
     }
 
+    if (is_builtin_type(func_name) || is_consensus_type(func_name))
+    {
+      // Replace the function call with a constant value. For example, x = int(1) becomes x = 1
```

Comment:
Why do we need to do this? Give some rationale in the comments.

Reply:
I've added a comment explaining it.